R/generate_simulated_data.R

```diff
@@ -364,7 +364,7 @@ generate_simulated_data <- function(r_in_weeks = # nolint
       mean = 0,
       sd = sqrt(scaling_factor) * sigma_eps
     )
-    n_t <- seq_len(ncol(log_r_site))
+    n_t <- ncol(log_r_site)
```

Comment:
I think we should use `n_weeks` here rather than redefining a new variable! https://github.com/CDCgov/ww-inference-model/blob/d255645e96f5d48b09f15addbd63d2c3db0a87b7/R/generate_simulated_data.R#L169